test/models/user_test.exs

```diff
@@ -34,15 +37,15 @@ defmodule Constable.UserTest do
     })
 
     refute changeset.valid?
-    assert changeset.errors[:email] == {"must be a member of thoughtbot.com", []}
+    assert changeset.errors[:email] == {"must be a member of #{@permitted_email_domain}", []}
   end
 
   test "create_changeset sets name from username only if the name is blank" do
-    changeset = User.create_changeset(%User{}, %{email: "foo@thoughtbot.com", name: "Real Name"})
+    changeset = User.create_changeset(%User{}, %{email: @valid_email, name: "Real Name"})
     assert changeset.changes[:name] == "Real Name"
 
     username = "foobar"
-    changeset = User.create_changeset(%User{}, %{email: username <> "@thoughtbot.com"})
+    changeset = User.create_changeset(%User{}, %{email: username <> "@#{@permitted_email_domain}"})
```

Comment:
It looks like we're interpolating and concat-ing here? How would you feel about only using interpolation? i.e. 
```ex
"#{username}@#{@permitted_email_domain}"
```